### ISSUE TITLE
Obsolete dialogs about joining factions in Cor Angar and Chronos, possibly Nefarius as well

### ISSUE THREAD

**Issue by MarkusRamikin** [AmProsius/gothic-1-community-patch#319]
I'm in chapter 3, after Yberion died. I'm a Fire Mage.

Issues:
- I can still ask Cor Angar how to become a Templar, repeatedly.
- I can still tell Chronos I want to become a magician (even though I am one), though only once.
- I can still tell Nefarius I want to be a Magician of Water. And then I can keep asking "Am I ready for the Circle of Water?". 
His answer "you must become a mercenary" makes it pretty clear this is obsolete too, meant for New Camp's Rogues. 
- Right now I can no longer ask Milten about becoming a magician's apprentice like him, but I do remember I could after taking the Oath and training magic circles. I don't know when it disappeared, but I do know it was later than made sense.

**Comment by AmProsius** [comment]
Thanks for reporting! The guild checks of the vanilla game seem to be quite inconsistent.
However, these are all individual bugs, we'll have to assign a separate issue to every single one.

**Comment by MarkusRamikin** [comment]
Come to think of it, I can tell Nefarius about wanting to become a Magician of Water even after becoming a Magician of Water.
